Headers/AppKit/NSTabViewController.h

```diff
@@ -0,0 +1,101 @@
+/* Definition of class NSTabViewController
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: 07-23-2020
```

Comment:
Could we switch to a more sensible date format here? MM-dd-yyyy may be popular in the US but isn't easy to understand for the rest of the world. The ISO standard would be yyyy-MM-dd, pretty far away what everybody is used to but a rather clear format. Or we stick to the old way dd-MM-yyyy is some form.

Reply:
Okay, that is fine.

Reply:
So the European format it is. ;)